osquery/tables/system/darwin/unified_log.mm

```diff
@@ -380,7 +380,8 @@ QueryData genUnifiedLog(QueryContext& queryContext) {
 
         Row r;
 
-        r["timestamp"] = BIGINT([[entry date] timeIntervalSince1970]);
+        r["timestamp"] = BIGINT(
+            static_cast<std::uint32_t>([[entry date] timeIntervalSince1970]));
```

Comment:
In the spirit of not losing information, do you think it makes sense to also pass a float through? (`timestamp_float` or something)

Reply:
Sure (I would call it `timestamp_double` to respect the true type).